sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/ManagementChannel.java

```diff
@@ -40,17 +52,38 @@
     public static final String MANAGEMENT_RESULT_LAST_ENQUEUED_TIME_UTC = "last_enqueued_time_utc";
     public static final String MANAGEMENT_RESULT_RUNTIME_INFO_RETRIEVAL_TIME_UTC = "runtime_info_retrieval_time_utc";
     public static final String MANAGEMENT_RESULT_PARTITION_IS_EMPTY = "is_partition_empty";
-
     // Well-known keys for management plane service requests.
     private static final String MANAGEMENT_ENTITY_TYPE_KEY = "type";
     private static final String MANAGEMENT_OPERATION_KEY = "operation";
     private static final String MANAGEMENT_SECURITY_TOKEN_KEY = "security_token";
 
+    public static final String REQUEST_RESPONSE_MESSAGES = "messages";
```

Comment:
Should these all go into ManagementConstants? There are two places where these strings live.